lang/en/cms.php

```diff
@@ -54,6 +56,7 @@
  or editable at the instance level, or be called upon to provide data. Some datasources are always included and do not appear in
  this list.';
 $string['cachedef_cms_content'] = 'CMS content';
+$string['visibility_updated'] = 'Visibility updated';
```

Comment:
Missing a lang string for the capability
```
Invalid get_string() identifier: 'cms:seeall' or component 'mod_cms'. Perhaps you are missing $string['cms:seeall'] = ''; in mod/cms/lang/en/cms.php?

    line 356 of /lib/classes/string_manager_standard.php: call to debugging()
    line 7414 of /lib/moodlelib.php: call to core_string_manager_standard->get_string()
    line 2653 of /lib/accesslib.php: call to get_string()
    line 2435 of /lib/accesslib.php: call to get_capability_string()
    line 118 of /admin/roles/classes/capability_table_base.php: call to get_capability_docs_link()
    line 135 of /admin/roles/classes/capability_table_with_risks.php: call to core_role_capability_table_base->display()
    line 708 of /admin/roles/classes/define_role_table_advanced.php: call to core_role_capability_table_with_risks->display()
    line 261 of /admin/roles/define.php: call to core_role_define_role_table_advanced->display()
```

Reply:
done